test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -243,13 +244,32 @@ class test_ucp_sockaddr : public ucp_test {
         UCS_TEST_MESSAGE << "server listening on " << m_test_addr.to_str();
     }
 
+    void start_listener(ucp_test_base::entity::listen_cb_type_t cb_type)
```

Comment:
why need this wrapper?

Reply:
To test custom listener logic (query conn_id)